src/Symfony/Bridge/Doctrine/Tests/Validator/Constraints/UniqueEntityValidatorTest.php

```diff
@@ -574,6 +574,11 @@ public function testValidateUniquenessWithCompositeObjectNoToStringIdEntity()
 
         $objectOne = new SingleIntIdNoToStringEntity(1, 'foo');
         $objectTwo = new SingleIntIdNoToStringEntity(2, 'bar');
+
+        $this->em->persist($objectOne);
```

Comment:
No the tests needs to be fixed,  the composite entity needs that its keys are flush first to initialize auto increment of their ids. See related failures in your PR.